src/managers/EmbraceSpanSessionManager/EmbraceSpanSessionManager.ts

```diff
@@ -208,6 +212,7 @@ export class EmbraceSpanSessionManager implements SpanSessionManagerInternal {
 
     this._sessionSpan.setAttributes({
       ...this._getPermanentAttributes(),
+      ...this._experienceManager.getExperienceAttributes(),
```

Comment:
Calling this twice: once at startSession and again at endSession. Only because you may want to read the attributes in the middle of a session. The second call is needed to update the lastActivity update.

Reply:
my mistake. I did this when I decided to report last activity time, but decided to only use it internally to determine which experience. There *are* no updates to the values within a session. I'll remove.